packages/react/src/internal/components/Status.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type {StoryObj} from '@storybook/react'
+import React, {useEffect, useState} from 'react'
+import {Status} from './Status'
+import {VisuallyHidden} from './VisuallyHidden'
+
+export default {
+  title: 'Private/Components/Status',
+  component: Status,
+}
+
+export const Default = () => {
+  const [message, setMessage] = useState('Default message')
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setMessage(`Last updated at ${new Date().toLocaleTimeString()}`)
+    }, 5000)
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
+
+  return <Status>{message}</Status>
+}
+
+export const VisuallyHiddenStory: StoryObj = {
+  name: 'VisuallyHidden',
+  render: () => {
+    return (
+      <>
+        <p>This is an example</p>
+        <VisuallyHidden>
```

Comment:
(non blocking) Ah, I was wondering if this should be a prop on the component or the default on the component. I guess we can find out once we start using it!

Reply:
Ideally, I want people to think of `Status` as `<div role="status">`. It won't do anything more or less than that role, but under-the-hood we make it work. So you can style it if you're using it visually, or if you want to hide then you use `VisuallyHidden`.

Reply:
> Ideally, I want people to think of Status as <div role="status">

Fair... but also do you see folks using it with visible elements? Maybe in Banner?

Reply:
@siddharthkp definitely, I think so.